packages/react-components/src/components/kpi/kpiBase.tsx

```diff
@@ -26,9 +29,15 @@ export const KpiBase: React.FC<KPIProperties> = ({
   return (
     <div className='kpi' style={{ fontSize: `${secondaryFontSize}px` }}>
       {showName && name}
-      {error && <ErrorBadge>{error}</ErrorBadge>}
+      {error && (
+        <Box margin={{ top: 's', bottom: 's' }}>
```

Comment:
Nit: Use `vertical` margin instead of `top` and `bottom`.

Reply:
done